<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="techniqum.Selenium.TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Submit Order Test(failed)">
    <classes>
      <class name="techniqum.Selenium.tests.StandAloneTest2">
        <methods>
          <include name="submitOrder" invocation-numbers="1"/>
          <include name="tearDown"/>
          <include name="launchApplication"/>
          <include name="orderHistoryTest"/>
        </methods>
      </class> <!-- techniqum.Selenium.tests.StandAloneTest2 -->
    </classes>
  </test> <!-- Submit Order Test(failed) -->
  <test thread-count="5" parallel="tests" name="Error Validations Test(failed)">
    <classes>
      <class name="techniqum.Selenium.tests.ErrorValidationsTest">
        <methods>
          <include name="tearDown"/>
          <include name="loginErrorValidation"/>
          <include name="launchApplication"/>
          <include name="checkOutErrorValidation"/>
        </methods>
      </class> <!-- techniqum.Selenium.tests.ErrorValidationsTest -->
    </classes>
  </test> <!-- Error Validations Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
